binding.gyp

```diff
@@ -29,8 +29,7 @@
             "src/js_email_password_auth.hpp",
             "src/js_api_key_auth.hpp",
             "src/js_auth.hpp",
-            "src/node/sync_logger.cpp",
-            "src/node/sync_logger.hpp",
+            "common/js_logger.hpp"
```

Comment:
Is the path correct? Shouldn't it be `src/common/js_logger.hpp`?

See also my comment below.

Reply:
it works on my machine : ). 

I can rename that if you want, it work both ways.